### ISSUE TITLE
Cannot update 18SE to 1.5.2

### ISSUE THREAD

**Issue by satnaam11** [FrSkyRC/ETHOS-Feedback-Community#3631]
Hello, My 18SE was at 1.5.1 and decided to update to 1.5.2 and the suite just hangs up on the writing firmware. I updated the 1.5.2 suite and everything gets updated except the firmware. I went back to Suite 1.5.1 and tried to reload 1.5.1 firmware and still I cannot update the firmware. Is there an issue with firmware  or issue with my hardware. Radio is working see the picture can't update 15.2 file. 
Thanks
satnaam

![18SE Nand memory](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/163063926/3e32741b-3627-4cd3-80e2-8e9b3249b17a)
[crash-20240311.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/14565636/crash-20240311.txt)
[debug-20240311.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/14565637/debug-20240311.txt)
I can write to NAND and read it. 
![Cant update 15 2](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/163063926/6fe12174-97aa-496e-931f-46f93c0e1568)

**Comment by Nicholas-Luoyi** [comment]
Does the radio start flashing itself if you unplug the USB after Suite hangs up?

**Comment by satnaam11** [comment]
No , the radio screen does not flash. It just displays Bootloader and PC screen shows Fatal Error Occurred. 
satnaam


Sent from Yahoo Mail for iPhone


On Monday, March 11, 2024, 8:25 PM, FrSky-Luo ***@***.***> wrote:




Does the radio start flashing itself if you unplug the USB after Suite hangs up?

**Comment by Nicholas-Luoyi** [comment]
Then the file copy operation should be failed before flashing starts.

**Comment by satnaam11** [comment]
The flashing never happens. When I remove the cable from the usb there is a message on PC fatal error occurred. On the radio screen the bootloader shows up. That’s it. Satnaam


Sent from Yahoo Mail for iPhone


On Monday, March 11, 2024, 8:42 PM, FrSky-Luo ***@***.***> wrote:




Then the file copy operation should be failed before flashing starts.

**Comment by salibaj2** [comment]
Had an issue updating my X18SE from 1.5.1 to 1.5.2. Suite kept waiting for the disk, until a `No disk error' popped up.

Then from the drop down list I selected to update the 'Firmware and i8n', hit update, and Tx updated to 1.5.2.

**Comment by satnaam11** [comment]
Thank you. Mine just gets stuck at writing and never starts the process. Just sits there. Don’t know what to do anymore. 


Sent from Yahoo Mail for iPhone


On Tuesday, March 12, 2024, 2:44 AM, salibaj2 ***@***.***> wrote:




Had an issue updating my X18SE from 1.5.1 to 1.5.2. Suite kept waiting for the disk, until a `No disk error' popped up.

Then from the drop down list I selected to update the 'Firmware and i8n', hit update, and Tx updated to 1.5.2.

**Comment by satnaam11** [comment]
![Success on 15 2](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/163063926/48374cd6-a17c-4811-af50-298a0bc6f48b)
![Tried to downgrade but no writing](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/163063926/ed22a325-28cd-4393-a680-cb414eb00d47)

Hello, I dived deep and here are the results. I went back to 14.5 and update, but still cannot update the firmware. I did the disk repair in the ETHOS suite 14.5 and ran fine. Update everything again to 14.5 and it ran through everything and passed. From there I loaded 15.2 suite and update the radio and success, but NAND drive is now FLASH drive ( is this OK? How do I name NAND?)I had a backup made during 15.1 and restored it and all looks ok. Checked one plane and all working as it should. Any comments from the team is welcome.
